docs/source/conf.py

```diff
@@ -2054,6 +2054,7 @@
     "PropModule",
     # torch.backends.cuda
     "cuBLASModule",
+    "MathSDPModule",
```

Comment:
Adding new entries in this file is not ok.
Please make sure that new APIs are either properly documented or not public.